src/presets.asm

```diff
@@ -318,10 +314,6 @@ if !FEATURE_PAL
 else
     JSL $A08A1E  ; Load enemies
 endif
-    JSL $82E071  ; Load room music
-    JSR $A12B    ; Play 14h frames of music
-    JSL $82E09B  ; Update music track index
-    JSL $82E113  ; RTL
```

Comment:
From what I can tell, we don't need to load room music because the preset already tells us what music to load and we already did it.